Partition_2/include/CGAL/Partition_2/Triangulation_indirect_traits_2.h

```diff
@@ -112,6 +112,13 @@ class Indirect_orientation_2
 };
 
 template <class Circulator>
+class Construct_circulator_2
+{
```

Comment:
[There](https://github.com/afabri/cgal/blob/67c531af5394e9faead89884a02cbc61e9b5c8c2/Partition_2/include/CGAL/Partition_2/Triangulation_indirect_traits_2.h#L146), just a consequence from requesting `Construct_point_2` in `TriangulationTraits_2`.